Source/Resources/Content/MSGraph/DeviceManagement/DeviceManagementScripts/Security - Set-PrivateNetConnectionProfile.json

```diff
@@ -7,7 +7,7 @@
       }
     }
   ],
-  "description": "Sets all network connection profiles as Private where the interface name is in a well known list of trusted networks (defined as $privateNetworks at the top of the script).\n\nThis allows corresponding Private profile Windows Firewall rules to be applied so that RDP, file sharing, discovery, etc. works on the LAN.",
+  "description": "What is it: Sets all network connection profiles as Private where the interface name is in a well-known list of trusted networks. \n Simeon Notes: This allows corresponding Private profile Windows Firewall rules to be applied so that RDP, file sharing, discovery, etc. works on the LAN.",
```

Comment:
@metamenos -- do we really think this should be in baseline? @jeff is wondering

**What does this do?**
Sets all network connection profiles as Private where the interface name is in a well-known list of trusted networks. 

**Why should you use this?**
This can help with troubleshooting scheduled tasks.

**What is the end-user impact?**
None. 

**Learn more**
[Enable Windows task scheduler history](https://medium.com/techygeekshome/enable-windows-task-scheduler-history-996a601a178c)

Reply:
I agree, this seems like one of those things that is very useful, IF you know what it does and how it works. Otherwise, could just be a question magnet.

Reply:
@metamenos so does that mean we should delete it from the baseline?